docs/namespace.md

```diff
@@ -0,0 +1,65 @@
+---
+id: namespace
+title: Customizing Namespace
+sidebar_label: Customizing Namespace
+---
+
+OpenEBS is usually deployed in the *default* namespace. You can deploy OpenEBS in another namespace  other than the *default* namespace.  To customize namespace to *openebs* instead of *default*,  you must change all the namespaces in the *openebs-operator.yaml* file to *openebs*. 
```

Comment:
you must replace all occurrences of the default namespace with namespace openebs.